### PR TITLE
Python3 doesn't have random.jumpahead

### PR DESCRIPTION
**Pull request by cybergrind**
Replace it with seed call

### CODE REVIEW

File: basictracer/util.py [hunk 1]
```diff
@@ -17,5 +16,5 @@ def generate_id():
     pid = os.getpid()
     if (_fork_guard_pid == 0) or (_fork_guard_pid != pid):
         _fork_guard_pid = pid
-        guid_rng.jumpahead(int(1000000 * time.time()) ^ pid)
+        guid_rng.seed(random.randrange(0, 0xFFFFFFFF))
```
Comment (yurishkuro) [inline basictracer/util.py]: wouldn't this produce identical result across all forks? The XOR with pid was adding uniqueness to each fork.

**Comment by yurishkuro** [comment]
If we want this change so that it works with python3, we should test it with 3. Can you add it to .travis.yaml matrix?

File: basictracer/util.py (hunk 1 above)
Comment (bensigelman) [inline basictracer/util.py]: Seed is fine, but the arg must incorporate pid (and should incorporate time IMO)

Comment (cybergrind) [inline basictracer/util.py]: Makes sense

**Comment by cybergrind** [comment]
I've added time + pid into seed.

Unfortunately protobuf tests aren't fine on python 3 due discrepancies in implementation of bytearray support
https://travis-ci.org/opentracing/basictracer-python/jobs/153806259
Looks like https://github.com/google/protobuf/issues/1173 caused it

I believe we can get rid of jumpahead now just to not break current applications that aren't rely on protobuf serialization and fix tests in separate branch. Btw it isn't look like it going to be fixed in nearest future in protobuf repo.

**Comment by yurishkuro** [comment]
OK fair enough, I will accept it. Thanks for checking against 3.x, I booked it as a separate issue maybe someone will care to fix: https://github.com/opentracing/basictracer-python/issues/16

**Pull request merged**